ngraph/test/util/test_case.hpp

```diff
@@ -182,6 +182,23 @@ namespace ngraph
                 EXPECT_TRUE(res);
             }
 
+            void run_with_tolerance_as_fp(const float tolerance = 1.0e-5f)
```

Comment:
Did you add this method because you faced with arithmetic float error?
What is the name of affected model(s)?

Reply:
I added this method, because there are ONNX Importer tests that are failed with the method run(), but the calculation error does not exceed 1e-5.